src/ripple/basics/impl/IOUAmount.cpp

```diff
@@ -49,6 +51,18 @@ IOUAmount::normalize()
         return;
     }
 
+    if (*stNumberSwitchover)
+    {
+        Number v{mantissa_, exponent_};
```

Comment:
It doesn't make much difference, but `v` can be `const` here.

Reply:
Good call, done.